.bazelci/presubmit.yml

```diff
@@ -10,248 +10,252 @@ buildifier:
 platforms:
```

Comment:
It seems like this file was reformatted which makes it a little hard to find diffs in. Couild you tag any changes you've made?

Reply:
Yes I'll fix the formatting ultimately.  PR is still in a draft state (probably should have made it a draft PR)